dev/docker/README.md

```diff
@@ -0,0 +1,86 @@
+Apache REEF (incubator) Docker Tests
+====================================
+
+[Apache REEF (incubator)](http://reef.incubator.apache.org/) is 
+an approach to simplify and unify the lower layers of big data systems
+on modern resource managers. This project maintains docker images
+to simulate the target underlying systems as similar as possible.
+
+Docker-based Test Cluster
+-------------------------
+
+| OS           | Java            | Hadoop Version            | Description   | alias     |
```

Comment:
Could you please add a new leftmost field called `Testing Environment` and fill in appropriate environments such as YARN/Mesos so that readers know which image is for testing which environment?